operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,110 @@
+By default, host and Kubernetes API audit logging are not enabled.
+
+### Procedure
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following options based on the installation status of the system:
+
+   * Install
+
+     To update the audit log settings during the installation, use one of the following options:
+
+     * **Option 1**
+
+       During the installation, audit logging is enabled or disabled by modifying the CSI settings.
+       To enable or disable audit logging, use the following flags with the `csi config init` command. For more information on using flags, see `csi config init -h`:
+
+       `ncn-mgmt-node-auditing-enabled`: Set to `true` to enable host logging or to `false` to disable host logging.
+
+       ```console
+       ncn# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
+       ```
+
+       `k8s-api-auditing-enabled`: Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+       ```console
+       ncn# csi config init --k8s-api-auditing-enabled=true [other config init options]
+       ```
+
+     * **Option 2**
+
+       Adjust the audit log settings by editing the `system_config.yaml` file.
+
+       View the current settings with the following command:
+
+       ```bash
+       ncn# cd /var/www/ephemeral/prep
+       ncn# grep audit system_config.yaml
+       ```
+
+       Example output:
+
+       ```text
+       k8s-api-auditing-enabled: false
+       ncn-mgmt-node-auditing-enabled: false
+       ```
+
+   * Post-install
+
+     If a system is installed, audit logging can be enabled through the `csi` command:
+
+     * `ncn-mgmt-node-auditing-enabled`
+
+       ```bash
+       ncn-m# export TOKEN=$(curl -k -s -S -d grant_type=client_credentials \
+       -d client_id=admin-client \
+       -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+       https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+       ncn-m# csi handoff bss-update-param --limit <mgmt-node-xname> --set ncn-mgmt-node-auditing-enabled=true
+       ```
+
+     * `k8s-api-auditing-enabled`
+
+       ```bash
+       ncn-m# export TOKEN=$(curl -k -s -S -d grant_type=client_credentials \
+       -d client_id=admin-client \
+       -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+       https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+       ncn-m# csi handoff bss-update-param --limit <mgmt-node-xname> --set k8s-api-auditing-enabled=true
+       ```
+
+     **NOTE**: If `csi` command is not installed, locate the `cray-site-init-*` rpm on the NCN master node and install it:
```

Comment:
Probably this should be added before this step. An optional step like "Install CSI on ncn-m001, if it is not already installed". People may see the CSI command to run and get stuck, not realizing the instructions they need are later on.